.circleci/config.yml

```diff
@@ -570,48 +637,17 @@ workflows:
       - upload-s3-sha:
           context: mattermost-ci-pr-builds-s3
           requires:
-            - build
+            - build-mattermost-server
+            - build-focalboard
       - upload-s3:
           context: mattermost-ci-pr-builds-s3
           requires:
-            - build
+            - build-mattermost-server
+            - build-focalboard
       - build-docker:
           context: matterbuild-docker
           requires:
             - upload-s3-sha
-      - test:
```

Comment:
@phoinixgrr  is it possible to remove only focalboard package tests from the pipeline. We will still need server tests.

Reply:
Aha, Thought we were disabling DB Tests for Mattermost-server also. 
Makes sense. Sure, will push a commit to fix this. 

Thanks @pfltdv

Reply:
Tests are now enabled with https://github.com/mattermost/mattermost-server/pull/20783/commits/6979e8685a47bc2fe0708195c7cb1075f1baabe4

Tests passing 
https://app.circleci.com/pipelines/github/mattermost/mattermost-server/36047/workflows/5172c280-48c4-4e1a-8b66-d7ea31213232